### DISCUSSION TITLE
Audio Conversion not possible: "[Errno 86] Bad CPU type in executable"

### DISCUSSION THREAD

**Discussion by TobiCR** [kaixxx/noScribe#61] [Q&A]
Mac Book Pro (2020)
Apple M1, 16GB
MacOS Sonoma 14.3
Macintosh HD

Dear community, 

Unfortunately, the audio conversion does not work on my Mac. It gives the following error: 
[Errno 86] Bad CPU type in executable: '/Applications/noScribe.app/Contents/Frameworks/ffmpeg'

I have tried several versions for the Apple processor, but I always get the same error message. 

I also tried running the Intel version via Rosetta 2, but that didn't work either. 

As I have only recently become an Apple user, I would need help, thank you in advance!

**Comment by gernophil** [comment]
Hey TobiCR,
could you give us some more info about your hardware? Haven't seen that error before. `ffmpeg` should run on both Intel and Apple Silicon CPUs.

**Reply by TobiCR** [reply]
sorry, I haven't finished the text yet and accidentally posted it too early. All the information is now included. Thanks for the quick feedback!

**Reply by gernophil** [reply]
Thanks for the info. That's really weird. Do you have Rosetta2 installed? Not that it should be necessary for the Apple Silicon version, but maybe we missed something. Do you have any experience with the terminal and can you simply start the bundled `ffmpeg` from terminal? Maybe also run `uname -m`?

**Reply by BabyFnord** [reply]
@TobiCR a related issue being solved here → https://github.com/kaixxx/noScribe/issues/78#issuecomment-2315101099
